routemaster/state_machine/tests/test_state_machine_api.py

```diff
@@ -0,0 +1,31 @@
+from unittest import mock
+
+from routemaster.state_machine import process_cron
+
+
+def test_handles_label_state_change_race_condition(app, create_deleted_label):
+    state_machine = app.config.state_machines['test_machine']
+    state = state_machine.states[1]
+
+    # Create a label which is not in the expected state. Doing this and then
+    # returning the affected label from the `get_labels` call is easier but
```

Comment:
I'd replace "but" with "and" here, if I'm reading this correctly.

Reply:
ffa70055a42965db27dd8174b9ad53b5caba9b24